lib/src/page_list_viewport_gestures.dart

```diff
@@ -454,57 +670,85 @@ class DeprecatedPanAndScaleVelocityTracker {
       : null;
 }
 
+double _repeatedSwipeVelocityIncreaseScalar(int numberOfRepeatedAcceleratedSwipes) {
+  // The function takes the number of the repeated swipes already considered in the repeated swipe acceleration sequence
```

Comment:
A description of what the function does should be provided as a Dart Doc above the function. Same for the function below this one.

Also, are these functions used by multiple classes? If not, please put them in the class that uses them. Moreover, if they're only called by one method, place them beneath that method so that the reader can quickly jump there and back, as necessary.